account_invoice_reminder/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+##############################################################################
+#
+#    OAC, Business Open Source Solution
+#    Copyright (C) 2013-2016 Open Architects Consulting SPRL.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```

Comment:
It's generally bad practice/manners to remove these blurbs from files. Is this accurate?

Reply:
this seems to be a module that was created by coop it easy. it’s strange to me that there is a copyright notice mentioning “open architects consulting”. @robinkeunen, do you know more about this?

i personally think that this blurb is useless in this context. the license property seems enough to me, but you know more about licensing than i do, i think.

Reply:
@huguesdk A blurb in every single file is generally a nice-to-have for portability reasons (super easy to copy files back and forth while retaining copyright and licensing information). But a blurb really only needs to contain these lines (example):

```python
# Copyright 2021 Coop IT Easy SCRLfs <https://coopiteasy.be/>
# Copyright 2019 Some Other Author
# ... etc
#
# SPDX-License-Identifier: AGPL-3.0-or-later
```

The `SPDX-License-Identifier` tag-value pair here can be replaced with a normal sentence like 'This software is licensed under AGPL 3 or later'. Anything more than the above is extraneous.

---

But when a blurb is already present, it's generally a bad idea to edit/remove it, unless it's obviously wrong. A lot of licences demand the exact reproduction of their blurbs. Example from MIT:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.\*

So generally it's a good idea to never-alter-only-append copyright-and-licensing blurbs.

\*: There is some ambiguity for interpretation here (you can move the blurb to a separate file and probably still comply), but never-alter-only-append as a rule of thumb basically guarantees that you can't mess this up.

Reply:
Leaving this as-is and adding it to my long-term todo list.